assets/webpack.config.js

```diff
@@ -44,35 +44,14 @@ module.exports = (env, options) => ({
         test: /\.(png|woff|woff2|eot|ttf|svg)$/,
         loader: "file-loader",
         options: {outputPath: "css", publicPath: "/css"}
-      },
-      {
```

Comment:
Notice that webpack checks for the `/umd` directory. UMD stands for Universal Module Definition. Seems that Bootstrap 3.4.1 doesn't have these module definitions. They were added in version 4, and now in version 5 they are gone. 

Here's the screenshot of the bootstrap `dist` folder:

![CleanShot 2024-11-30 at 21 15 20@2x](https://github.com/user-attachments/assets/e49918f7-a429-4ba4-816b-8e4e5bdc8cc6)

As of now, it doesn't affect bundled files. Though, we can remove `/umd` postfix to inject jquery here, but it seems a more reliable way of injection is to use `ProvidePlugin`. The plugin is also very close to what `esbuild` does with its injections.